apps/forwarding/signals.py

```diff
@@ -0,0 +1,102 @@
+"""
+Signal handlers for forwarding app.
+
+
+How ForwardingConfig scheduling works
+-------------------------------------
+
+When a ``ForwardingConfig`` instance is created with a ``Schedule``:
+
+1. The ``post_save`` signal is triggered.
```

Comment:
Django signals make me queasy. There are other ways database records can get updated which will not trigger the signal (bulk updates, updates by tools other than the Django ORM). I think it's fine to use a signal to create a responsive UI experience, but not great if that's the only way these things are kept in sync. What do you think of also using a periodic task to sync schedules in case of updates that do not trigger the signal?

Reply:
That sounds wise. What kind of period seems reasonable to you?